packages/react-liveness/src/components/FaceLivenessDetector/service/types/liveness.ts

```diff
@@ -1,3 +1,5 @@
+import { CredentialProvider } from '@aws-sdk/types';
```

Comment:
This seems to be deprecated? here: https://github.com/aws/aws-sdk-js-v3/blob/dac4b45efca3057042a528e520da35adafe930e3/packages/types/src/credentials.ts#L18